test/crowdsale/Crowdsale.test.js

```diff
@@ -15,67 +16,118 @@ contract('Crowdsale', function ([_, investor, wallet, purchaser]) {
   const value = ether(42);
   const tokenSupply = new BigNumber('1e22');
   const expectedTokenAmount = rate.mul(value);
+  const ZERO_ADDRESS = '0x0000000000000000000000000000000000000000';
 
-  beforeEach(async function () {
-    this.token = await SimpleToken.new();
-    this.crowdsale = await Crowdsale.new(rate, wallet, this.token.address);
-    await this.token.transfer(this.crowdsale.address, tokenSupply);
+  it('requires a valid token', async function () {
+    await assertRevert(
+      Crowdsale.new(rate, wallet, ZERO_ADDRESS)
+    );
   });
 
-  describe('accepting payments', function () {
-    it('should accept payments', async function () {
-      await this.crowdsale.send(value);
```

Comment:
Instead of removing this line we should have a separate test for receiving payments through the fallback function.

Reply:
That is precisely what I did: check the 'bare payments' `describe `. Having more than one test in a single `it `is a bad practice imo.